src/templates/static/get_started/binary_options.jsx

```diff
@@ -166,7 +166,7 @@ const BinaryOptions = () => {
                             <Step header={it.L('Make your trade')}      text={it.L('Buy the contract or re-define your position.')} />
                         </div>
 
-                        <h3 className='secondary-color'>{it.L('Step 1: Define your position')}</h3>
+                        <HeaderSecondary header={it.L('Step 1: Define your position')} />
```

Comment:
any reason why we are changing this to HeaderSecondary? (it will add a separator line before the header tag)

Reply:
oh i see, i thought it just rendered `h3` at the time of change (all others steps used `HeaderSecondary` so changed it for consistency)